packages/clay-form/src/MultiSelectInput.tsx

```diff
@@ -155,7 +185,16 @@ const ClayMultiSelect: React.FunctionComponent<IProps> = ({
 
 		const pastedItems = pastedText
 			.split(',')
-			.map(item => item.trim())
+			.map(itemLabel => {
+				itemLabel = itemLabel.trim();
+
+				const newItem: Item = getSourceItemByLabel(itemLabel) || {
```

Comment:
Just out of curiosity, why do we need to get the item from the array of `sourceItems`?

Reply:
'cause most of the times the value of an item is no the same as the label, so if is an item present in the autocomplete items we need to get the correct value from there. If not present we just make the value the same as the label.

Reply:
Hm yeah, well thought.